install/helm/agones/templates/service/allocation.yaml

```diff
@@ -161,8 +184,19 @@ spec:
         - name: FEATURE_GATES
           value: {{ .Values.agones.featureGates | quote }}
         ports:
+        {{- if .Values.agones.allocator.service.http.enabled }}
         - name: https
```

Comment:
I think that `name` should also be a changeable attribute if we're already changing the port values.  There are some use cases where changing the port to `80` and leaving the name as `https` would be disadvantageous.  If not, then perhaps we should statically name this to just be `http` to be consistent with the nomenclature of the helm values label.

Reply:
Done.